### PR TITLE
Stream defaults and white space corrections in xml file

### PR DESCRIPTION
**Pull request by mark-petersen**
Fix indenting, space to tabs, change variables in a default MPAS stream. This affects MPAS auto-generated streams file, but not anything in E3SM.

### CODE REVIEW

**Comment by mark-petersen** [comment]
@pwolfram visual inspection is fine. This is just clean-up.

File: src/core_ocean/analysis_members/Registry_high_frequency_output.xml [hunk 1]
```diff
@@ -216,92 +216,95 @@
...
+				/>
+		</var_array>
 	</var_struct>
 	<streams>
 		<stream name="highFrequencyOutput" type="output"
 				mode="forward;analysis"
 				filename_template="analysis_members/highFrequencyOutput.$Y-$M-$D.nc"
 				filename_interval="01-00-00_00:00:00"
-				output_interval="00-00-01_00:00:00"
+				output_interval="00-00-05_00:00:00"
```
Comment (pwolfram) [inline src/core_ocean/analysis_members/Registry_high_frequency_output.xml]: @mark-petersen, is this change intentional?  This isn't a whitespace chagne.

File: src/core_ocean/analysis_members/Registry_high_frequency_output.xml [hunk 2]
```diff
@@ -216,92 +216,95 @@
...
 				reference_time="0001-01-01_00:00:00"
 				packages="highFrequencyOutputAMPKG"
 				clobber_mode="truncate"
 				runtime_format="single_file">
 			<stream name="mesh"/>
 			<var name="xtime"/>
+			<var name="kineticEnergyAtSurface"/>
+			<var_array name="activeTracersAtSurface"/>
```
Comment (pwolfram) [inline src/core_ocean/analysis_members/Registry_high_frequency_output.xml]: I agree we need these in-- but again, not a white space change.  Probably should highlight this in the PR description.

Comment (mark-petersen) [reply src/core_ocean/analysis_members/Registry_high_frequency_output.xml]: Right. Updated PR name and description.

File: src/core_ocean/analysis_members/Registry_high_frequency_output.xml (hunk 1 above)
Comment (mark-petersen) [reply src/core_ocean/analysis_members/Registry_high_frequency_output.xml]: Right. Updated PR name and description.
